tests/aws-spawn-test.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+set -ex
+# this is a fairly exhaustive test and will take some time
+# spawn all reasonable "platform" specifications
+function test() {
+  platform=$1
+  version=$2
+
+  for role in client hub; do
+    cf-remote spawn --count 1 --platform "$platform-$version" --role "$role" --name "$platform-$version-$role"
```

Comment:
Can this leave spawned AMI's if script fails somehow? 💸

Reply:
I clean up one at a time and this script is very "manual" as-is, not intended to be automated though I suppose it could be if we ran it in a place that had AWS privs/creds.

I will add a destroy all to a trap. 👍